.vib/drupal/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,32 @@
+const COMMAND_DELAY = 2000;
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
+}
+
+Cypress.Commands.add('iframe', { prevSubject: 'element' }, $iframe => {
```

Comment:
I added this function to handle the iframe used by the application for adding the article text. Cypress is not very straightforward with how it handles Iframes it seems.

Reply:
I'm not sure if there is a need to add the whole iframe logic to just add the article's text (body), which is not a mandatory field. Having to fill the title and uploading an image would be testing enough so if typing the body requires extra logic I think it is not worthwhile. WDYT @mdhont @joancafom?

Reply:
My idea for adding it was that if we only test the main functionality then let's make sure at least that the common steps are tested.